### ISSUE TITLE
Invalid order in analysis of force order

### ISSUE THREAD

**Issue by kondziu** [PRL-PRG/R-dyntrace#103]
Function:
```
f<-function (x, y) if (x > 0) f(x - 1, y) else y + 1
```
Force order:
```
x= < x! < y!
```
Correct force order:
```
x! < x= < y!
```
Because (excerpt):
```
>> call closure name=f call_id=4 function_id=0 parent_id=2 location=<console> compiled=false arguments={x:3,y:4}
>> call special name=if call_id=5 function_id=1 parent_id=0 location=<unknown> compiled=false
>> call built-in name=> call_id=6 function_id=2 parent_id=0 location=<unknown> compiled=false
>> force promise name=x id=3 in_call=6 from_call=4                                                                       ##### x!
>> call built-in name=- call_id=7 function_id=3 parent_id=140546524754186 location=<unknown> compiled=false
<> lookup promise name=x id=1 in_call=7 from_call=1                                                                      ##### x=
<< call built-in name=- call_id=7 function_id=3 parent_id=94408183911128 location=<unknown> compiled=false
<< force promise name=x id=3 in_call=6 from_call=4
<< call built-in name=> call_id=6 function_id=2 parent_id=94408180444672 location=<unknown> compiled=false
...
```

**Comment by kondziu** [comment]
I'll get rid of this ordering here, will relegate this to R post-processing.

**Comment by kondziu** [comment]
This will be fixed when #102 is fixed, basically.

**Comment by kondziu** [comment]
Order analysis moved to R.
